frontend/packages/data-portal/app/components/Run/ViewerPage.tsx

```diff
@@ -454,7 +556,7 @@ function ViewerPage({ run, tomogram }: { run: any; tomogram: any }) {
         setShareClicked(true)
       })
       .catch((err) => {
-        console.error('Failed to copy URL: ', err)
+        throw new Error(`Failed to copy share link: ${err}`)
```

Comment:
I think it would be safer for testing purpose to force the `throw new Error` as it can bubble up until higher level component in react and crash the full app if there is no error boundaries management.

Reply:
Ah sorry, the change I made here doesn't make much sense. We had a lot of linting messages so I probably went a bit fast on this one